src/plugins/blackbox/public/webcomponents/orov-blackbox-downloader.html

```diff
@@ -211,6 +217,9 @@
             this.eventEmitter.emit('plugin-blackbox-export',options);
           }
         },
+        showNativeExport: function(){
```

Comment:
The system checks for the object that indicates the native file system feature is available at which point it hides the old export options and dispalys the single video export button.